app/templates/skeleton/app/modules/configuration/identity/identity-config.js

```diff
@@ -9,6 +9,7 @@ angular.module('configuration.identity', ['configuration.rest', 'ngCookies', 'ui
         });
 
         $scope.signOut = function () {
+            $resource(restConfigService.getLogoutOperation()).delete();
```

Comment:
I've refactored the identiy-config and the authentication-config so the flow a little better now.  Should be in the next pull request.